lib/paranoia_helper.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+module ParanoiaHelper
```

Comment:
I'd probably call this something else. `ParanoiaHelper` isn't very immediate as to its purpose.